lib/taskana-data/src/main/resources/sql/test-data/history-event.sql

```diff
@@ -1,4 +1,4 @@
-INSERT INTO HISTORY_EVENTS (ID,BUSINESS_PROCESS_ID, PARENT_BUSINESS_PROCESS_ID, TASK_ID, EVENT_TYPE, CREATED, USER_ID, DOMAIN, WORKBASKET_KEY, POR_COMPANY, POR_SYSTEM, POR_INSTANCE, POR_TYPE, POR_VALUE, TASK_CLASSIFICATION_KEY,
```

Comment:
Haven't we agreed to split this into two files?

Reply:
done